roles/swapfile/tasks/main.yml

```
@@ -33,7 +33,7 @@
 
 - name: "activate {{ swapfile_path }}"
   command: "swapon {{ swapfile_path }}"
-  when: swapfile_active | failed
+  when: swapfile_active.failed | bool
```

I think what @evgeni suggested was:
```suggestion
  when: swapfile_active is failed
```

---

Correct